analytics.go

```diff
@@ -20,14 +20,6 @@ const Version = "3.0.0"
 // Endpoint for the Segment API.
 const Endpoint = "https://api.segment.io"
 
-// DefaultContext of message batches.
-var DefaultContext = map[string]interface{}{
-	"library": map[string]interface{}{
-		"name":    "analytics-go",
-		"version": Version,
-	},
-}
```

Comment:
Oh interesting, you mean by setting values of the `DefaultContext` variable then it gets picked up automatically?

Global state is usually not a good practice, how about making this part of the client configuration when it's instantiated?